packages/api/.eslintrc

```diff
@@ -19,6 +19,7 @@
     // Though we aren't requiring JSDoc blocks to be present, if they are they should be properly
     // formatted.
     "jsdoc/require-jsdoc": "off",
+    "jsdoc/tag-lines": "off",
```

Comment:
Disabling this because I think the `@example` docs look cleaner with a space between them.

![Screen Shot 2022-03-17 at 5 56 36 PM](https://user-images.githubusercontent.com/33762/158917360-531f377c-6b76-4653-bad1-8946e8fa93d0.png)